File: examples/functions/assets-spa-fallback/php/api-profile/handler.php
<?php

function handler($event) {
    return [
        'status' => 200,
        'headers' => ['Content-Type' => 'application/json'],
        'body' => json_encode([
            'runtime' => 'php',
            'title' => 'PHP profile endpoint',
            'summary' => 'Classic API route living beside the SPA shell.',
            'path' => '/api-profile',
        ]),
    ];
}

File: examples/functions/assets-spa-fallback/lua/api-flags/handler.lua
local cjson = require("cjson.safe")

function handler(_event)
  return {
    status = 200,
    headers = { ["Content-Type"] = "application/json" },
    body = cjson.encode({
      runtime = "lua",
      title = "Lua feature flags endpoint",
      summary = "In-process Lua stays available while the browser router handles deep links.",
      path = "/api-flags",
    }),
  }
end
